samples/language/WhileLoops.qs

```diff
@@ -0,0 +1,18 @@
+/// # Sample
+/// While Loops
+///
+/// # Description
+/// While-loops are a basic looping looping structure in Q# for repeatedly
```

Comment:
nit: similar to Alex's comment, here it should probably be "While loops" instead of "While-loops".